src/commands/internal/create-changelog.ts

```diff
@@ -101,26 +112,51 @@ export async function getChangelogText(mode: string, startRef: string): Promise<
     process.exit(2);
   }
 
+  const previousPreVersions = getPreviousPreVersionsInReleaseChannel(nextVersion);
+  const shouldShowPreviousPreVersions = previousPreVersions.length > 0;
+  let currentPreviousPreVersion = shouldShowPreviousPreVersions ? { tag: nextVersion, date: new Date() } : null;
+  let currentPreviousPreVersionIndex = -1;
+
   const mergedPullRequestsText = mergedPullRequests
     .map((pr: PullRequest): string => {
-      const mergedAt = moment(pr.mergedAt).format('YYYY-MM-DD');
+      let text = '';
+      const mergedAtDate = moment(pr.mergedAt);
+      const mergedAtString = mergedAtDate.format('YYYY-MM-DD');
+      const introducedInCurrentPreviousPreVersion =
+        currentPreviousPreVersion != null && mergedAtDate.isBefore(currentPreviousPreVersion.date);
+
+      if (introducedInCurrentPreviousPreVersion) {
```

Comment:
Es wurde ja explizit geprüft, dass der timestamp **vor** der aktuellen previousPreVersion ist:

```suggestion
      const introducedInPreviousPreVersion =
        currentPreviousPreVersion != null && mergedAtDate.isBefore(currentPreviousPreVersion.date);

      if (introducedInPreviousPreVersion) {
```

Reply:
Alle PRs sind in "previous pre versions" introduced worden. 

Hier iterieren wir versetzt über die Liste an "previous pre versions" und haben eine "current previous pre version". Klingt komisch, ist aber so.

Reply:
Klar, aber mit `mergedAtDate.isBefore(currentPreviousPreVersion.date);` prüfst du, dass der PR in einer preVersion liegt, der **previous** zur **current** preVersion ist. Deswegen ist der Name introducedIn**Current**PreviousPreVersion falsch - oder hab ich grad einen Knoten im Kopf? 🤔

Reply:
Wenn der PR der erste **vor** dem Zeitpunkt der aktuell festgehaltenen Pre-Version (current) ist, dann ist die Pre-Version jene, in welcher der PR erstmals veröffentlicht wurde.

Dann schreiben wir diese Pre-Version als Zwischenüberschrift und ändern die festgehaltene Pre-Version, um das Spielchen von vorne anzufangen.

Evtl. kommt der Knoten auch daher, dass wir rückwärts durch die Zeit laufen?

Reply:
>Wenn der PR der erste vor dem Zeitpunkt der aktuell festgehaltenen Pre-Version (current) ist, dann ist die Pre-Version jene, in welcher der PR erstmals veröffentlicht wurde.

Ah! 💡  Jetzt wird ein Schuh draus. Natürlich gehört der PR noch zu der aktuellen Version - ich hatte die ganze Zeit gedacht, dass es der erste der nächsten previousPreVersion ist. Vielen Dank, dass du dir die Zeit genommen hast meinen Knoten zu lösen 🙈